src/Directory.UnoMetadata.targets

```diff
@@ -0,0 +1,28 @@
+<Project>
+	<PropertyGroup>
+		<Authors Condition="'$(Authors)'=='' or '$(Authors)'=='$(PackageId)' or '$(Authors)'=='$(AssemblyName)'">Uno Platform</Authors>
+		<Copyright Condition="'$(Copyright)'==''">Copyright (C) 2015-$([System.DateTime]::Now.ToString(`yyyy`)) Uno Platform inc. - all rights reserved</Copyright>
+		<RepositoryUrl  Condition="'$(RepositoryUrl)'==''">$(BUILD_REPOSITORY_URI)</RepositoryUrl>
+		<Product  Condition="'$(Product)'==''">$(AssemblyName) ($(TargetFramework))</Product>
+		<PackageLicenseExpression  Condition="'$(PackageLicenseExpression)'==''">Apache-2.0</PackageLicenseExpression>
+		<DefaultLanguage Condition="'$(DefaultLanguage)'==''">en-US</DefaultLanguage>
+	</PropertyGroup>
+
+	<Target Name="DownloadAndSetPackageIcon" AfterTargets="Build"  Condition="'$(PackageIcon)'==''">
+		<PropertyGroup>
+			<IconUrl>https://nv-assets.azurewebsites.net/logos/uno.png</IconUrl>
```

Comment:
There is no MSBuild property named `IconUrl`, so this has no effect. The correct property name is `PackageIconUrl`, but that's deprecated and we shouldn't use it.

We should do it like this:

https://github.com/unoplatform/uno/blob/f79cc9c92c11e65aba850452b993bbe62328eb9e/src/Uno.UI.Runtime.Skia.Tizen/Uno.UI.Runtime.Skia.Tizen.csproj#L11


https://github.com/unoplatform/uno/blob/f79cc9c92c11e65aba850452b993bbe62328eb9e/src/Uno.UI.Runtime.Skia.Tizen/Uno.UI.Runtime.Skia.Tizen.csproj#L21-L23